README.md

```diff
@@ -1 +1,51 @@
-# ironfoudation-team-project
+# WOOF application 
+
+⚠️ The app is under active development.
+
+## Current dev environment 
+
+| Dev environment| Ref link |
+| --- | -- |
+| ![macOS](https://img.shields.io/badge/macOS-13.3+-blue) | | 
+| ![Xcode](https://img.shields.io/badge/Xcode-14.3-red) | |
+| ![Swift](https://img.shields.io/badge/Swift-5.8-orange) | |
+| ![SDKs for iOS](https://img.shields.io/badge/SDKs%20for%20iOS-15%2B-lightgrey) | |
+| ![SwiftFormat](https://img.shields.io/badge/SwiftFormat-0.51.7-yellow) | [Link](https://github.com/nicklockwood/SwiftFormat) |
+| ![SwiftLint](https://img.shields.io/badge/SwiftLint-0.51.0-green)| [Link](https://github.com/realm/SwiftLint) |
+
+## Table of contents
+* [General info](#general-info)
+* [Installation](#installation)
+* [Authors](#authors)
+
+## General info
+
+The repo contains the source code of the `Woof` application.
+The `Woof` application is an iOS app that provides the opportunity to connect dog owners and pet sitters to build trusted relationships between them.
+You can use the app in 2 modes: like a pet owner or like a pet sitter.
+As a pet owner, you can find a sitter for your pet(s) and manage a schedule of your pets' walks, which makes the process of caring for your lovely doggies easier.
+As a pet sitter, you can provide the service of walks using your own schedule, prices, and other requirements, if needed.
+
+## Installation
+
+**Pre-requisites**: 
+- installed Xcode with the version specified under the [Current development Environment](https://github.com/ios-course/ironfoudation-team-project/blob/34d602116ed0d8664f4cfb4a11563b4a3dc1f0c2/README.md#current-dev-environment).
```

Comment:
You have a link to the specific commit please read the [documentation](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files). Please have a link to the current commit and use anchors as you have done in the table of contents:

```suggestion
- installed Xcode with the version specified under the [Current development Environment](#current-dev-environment).
```

Reply:
Thanks! My bad. Fixed it.